dependencies/dockerfiles/maxtext_runner.Dockerfile

```diff
@@ -16,7 +16,6 @@ WORKDIR /deps
 # Copy assets separately
 COPY src/MaxText/assets/ "${MAXTEXT_ASSETS_ROOT}"
 COPY src/MaxText/test_assets/ "${MAXTEXT_TEST_ASSETS_ROOT}"
-COPY generated_requirements .
```

Comment:
Was this removed intentionally? I guess it just isn't used?

Reply:
Also would like to know. Thanks!

Reply:
That whole folder has been moved under dependencies so it's redundant and incorrect to retain.